test/src/source-map-cache.js

```diff
@@ -35,10 +35,16 @@ try {
 
 var sourceMapCache = new SourceMapCache()
 _.forOwn(covered, function (fixture) {
-  sourceMapCache.add(fixture.relpath, fixture.contentSync())
+  var source = fixture.contentSync()
+  var sourceMap = convertSourceMap.fromSource(source) || convertSourceMap.fromMapFileSource(source, fixture.relpath)
+  if (sourceMap) {
+    sourceMapCache.addMap(fixture.relpath, sourceMap.sourcemap)
+  }
 })
 
-var coverage = ap(require('../fixtures/coverage'))
+var getReport = function () {
+  return _.cloneDeep(require('../fixtures/report'))
```

Comment:
I think if you switch this line to:

```js
return ap(_.cloneDeep(require('../fixtures/report')))
```

I wouldn't complain if istanbul only stored posix-style paths, it's a bit of an edge-case though -- both `\`, and `/` work on Windows, it's just biting us in the lookups we're doing for tests.